app/assets/javascripts/homePage.js

```diff
@@ -4,5 +4,6 @@
 //= require_tree ./shared
 //= require_tree ./sidebar
 //= require_tree ./profile
+//= require_tree ./homepage
 
-app = angular.module("HomePage", ["ui.bootstrap", "ngRoute", "Profile", "Sidebar"])
+app = angular.module("HomePage", ["ui.bootstrap", "ngRoute", "SharedServices", "Profile", "Sidebar", "Conversation"])
```

Comment:
`SharedServices` and the `//= require_tree ./homepage` were removed during refactoring in 46d2683.  Are they necessary here?